### DISCUSSION TITLE
Dark mode for lists no longer works (what?)

### DISCUSSION THREAD

**Discussion by qbjl** [ScratchAddons/ScratchAddons#2398] [Help] [answered]
I've checked my settings but when I see lists they're always the eye-blinding white...
What happened??

**Comment by mxmou** [comment, marked as answer]
Based on user feedback, we decided that dark mode should not affect the stage. Maybe it could be an option?

**Reply by qbjl** [reply]
yes

**Comment by TheColaber** [comment]
The stage is also eye-blinding white unless you change the background. However if scratchaddons changed the list color, everyone else playing your dark background project would have an eye-blinding list, just like you if they didn't have scratch addons. So what is the solution? 
1. Make your project completely eye-blinding
2. Don't use the eye-blinding list.
